jupyterlab/docker-compose.yml

```
@@ -0,0 +1,14 @@
+services:
+  app_proxy:
+    environment:
+      APP_HOST: jupyterlab_server_1
+      APP_PORT: 8888
+      PROXY_AUTH_ADD: "false"
+
+  server:
+    image: quay.io/jupyter/base-notebook:latest@sha256:929c9adc94c4ad2b729e5b0dbddd3677c36c4ab6391d6e58b6e666c39aa65673
```

Can we use this tag also?

```suggestion
    image: quay.io/jupyter/base-notebook:lab-4.4.10@sha256:e68bcffc0e065e192f8aa1935ef04513adbbe9e250226d2189fa018b39f66980
```